### ISSUE TITLE
Unversioned third party resources break openshift api

### ISSUE THREAD

**Issue by karmab** [openshift/origin#16860]
[provide a description of the issue]
after injecting thirdparty resources with an incorrect name ( say karmalabs.local instead of stable.karmalabs.local ), openshift api stops to work properly, for instance oc get scc wil fail with the following  "the server doesn't have a resource type "scc"", while oc get securitycontextconstraints works

##### Version
[provide output of the `openshift version` or `oc version` command]
oc v3.7.0-alpha.1+fdbd3dc
kubernetes v1.7.0+695f48a16f
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://127.0.0.1:8443
openshift v3.7.0-alpha.1+fdbd3dc
kubernetes v1.7.0+695f48a16f

##### Steps To Reproduce

1. [step 1]
create third party resource

```
cat <<END | oc create -f -
apiVersion: extensions/v1beta1
kind: ThirdPartyResource
metadata:
  name: frout.karmalabs.local
description: "A specification of a Frout"
versions:
- name: v1
END
```

2. [step 2]
create an element of this third party

```
cat <<END | oc create -f -
apiVersion: "karmalabs.local/v1"
kind: Frout
metadata:
  name: frout1
size: big
END
```
****

##### Current Result
error: unable to recognize "frout1.yml": no matches for karmalabs.local/, Kind=Frout

##### Expected Result
frout "frout1" created

**Comment by karmab** [comment]
any updates ? also saw similar behaviour when testing and old version of istio(0.1.6)

**Comment by RHodzelmans** [comment]
Indeed I see a similar issue with:

oc v3.6.173.0.21
kubernetes v1.6.1+5115d708d7
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server <hidden, but on AWS>
openshift v3.6.173.0.49
kubernetes v1.6.1+5115d708d7

**Comment by karmab** [comment]
closing this as i m not getting the issue anymore with openshift 3.7.0. and fission 0.4.0
deploys perfectly fine

**Comment by smarterclayton** [comment]
Good to hear

On Dec 4, 2017, at 1:44 PM, Karim Boumedhel <notifications@github.com>
wrote:

Closed #16860 <https://github.com/openshift/origin/issues/16860>.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/issues/16860#event-1371235767>, or mute
the thread
